tests/serializers/base_serializer_test.py

```diff
@@ -1,11 +1,6 @@
-from expects import expect, have_keys
+from expects import expect, have_keys, equal
 
-from app.serializers.base_serializer import (
-    BaseSerializer,
-    camel_case,
-    field,
-    fields,
-)
+from app.serializers.base_serializer import BaseSerializer, camel_case
```

Comment:
Why are we importing `camel_case` from the BaseSerializer? I guess it kind of makes sense since that might be the only file that really needs it, but it seems more general than that. Would it make sense to move it to a helper file?

Reply:
Yeah, it should go in a `string_utils` module ... I was just trying to keep the scope small.